.i18nrc.json

```diff
@@ -42,6 +43,14 @@
     "x-pack/plugins/infra/public/graphql/types.ts",
     "x-pack/plugins/infra/public/utils/loading_state/loading_result.ts",
     "x-pack/plugins/infra/server/graphql/types.ts",
-    "x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts"
+    "x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts",
```

Comment:
is there a better way to do that? so we do not pollute this file. Can we have our own `.i18nrc.json` in our project

Reply:
Kibana currently does not have this capability. I checked in the source code where it uses this file as well as tested it locally to see if it worked or not.

This would be work in kibana core team for this feature to be added.

Reply:
@FrankHassanabad can you please help us to understand what's the reason behind excluding that many files? I know all previously excluded files where because of bug in Babel TS plugin we use for i18n parsing (usually because of valid, but weird TS syntax like args de-structuring in interface definition), but not sure what is reason in your case.